features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/Resources.jsx

```diff
@@ -136,24 +137,26 @@ class Resources extends React.Component {
                             </Typography>
                         )}
                 </div>
-                <div className={classes.root}>
-                    <div className={classes.contentWrapper}>
-                        {Object.keys(paths).map((key) => {
-                            const path = paths[key];
-                            return (
-                                <div className={classes.root}>
-                                    <Typography className={classes.heading} variant='body1'>
-                                        {key}
-                                    </Typography>
-                                    {Object.keys(path).map((innerKey) => {
-                                        return CONSTS.HTTP_METHODS.includes(innerKey) ?
-                                            <RenderMethod method={innerKey} /> : null;
-                                    })}
-                                </div>
-                            );
-                        })}
+                <Box p={1}>
+                    <div className={classes.root}>
+                        <div className={classes.contentWrapper}>
+                            {Object.keys(paths).map((key) => {
```

Comment:
I think we can use `Object.entries()` with `for ... of`  instead of a double loop.

Reply:
Let's do a code refactoring later, I just reorganized the existing code for layout changes.